client/ayon_traypublisher/plugins/publish/collect_clip_instances.py

```diff
@@ -33,4 +49,37 @@ def process(self, instance):
                 instance.context.data["editorialSourcePath"])
             instance.data["families"].append("trimming")
 
+        if repres := instance.data.pop("prep_representations", None):
```

Comment:
Could this condition be reversed to avoid the indentation?
```suggestion
        repres = instance.data.pop("prep_representations", None):
        if not represe:
            return
```

Or process it in different method
```suggestion
        if repres := instance.data.pop("prep_representations", None):
            self._convert_prepared_representations(instance, repres)
```